src/i18n.js

```diff
@@ -0,0 +1,61 @@
+// https://vue-i18n.intlify.dev/
+
+import { createI18n } from 'vue-i18n';
+
+const messages = {
+  en: {
+    global: {
+      signup: 'Sign Up',
+      login: 'Log In',
+      logout: 'Log Out',
+      rules: 'Rules',
+      play: 'Play',
+      stats: 'Stats',
+      username: 'Username',
+      password: 'Password',
+      collapseMenu: 'Collapse Menu',
+    },
+    login: {
+      noAccount: `Don't have an account?`,
+      haveAccount: `Already have an account?`,
+      title: 'What is Cuttle?',
+      quote: '"Cuttle is a sharp, fast game built entirely on excellent mechanics. It is the sort of game - had I known about it in college - I would have worn decks ragged through play."',
+      quoteCite: 'Richard Garfield - Creator of Magic: The Gathering',
+      paragraph1: `Cuttle is a 2 player battle card game played with a standard 52-card deck of cards. It has the strategic nuance of trading card games like Magic, with the elegant balance of a standard deck--and you can play it for free! Test your mettle in the deepest cardgame under the sea!`,
+      paragraph2: `Be the first to score 21 points in this explosive battle of wits. Mount a valiant offense while disrupting your opponent with dastardly tricks. Do you have what it takes to become the Lord of the Deep?`,
```

Comment:
This is my least favorite part of this setup and needs to change. I don't have a ton of experience with large setups of translating a lot of text, but when we have multiple paragraphs of the "same content", there has to be a better way to enable proper spacing and still put the content on the page.

Maybe some sort of markdown component we can pass markdown in to? Something like this I would imagine for a string called `loginPageContent` or whatever:

```
## What is Cuttle?

Cuttle is a 2 player battle card game played with a standard 52-card deck of cards. It has the strategic nuance of trading card games like Magic, with the elegant balance of a standard deck--and you can play it for free! Test your mettle in the deepest cardgame under the sea!

Be the first to score 21 points in this explosive battle of wits. Mount a valiant offense while disrupting your opponent with dastardly tricks. Do you have what it takes to become the Lord of the Deep?
```

Where markdown would automatically convert the `##` in to a heading, and then automatically wrap the paragraphs in p tags. It would also make contributing translations much easier imo.